app/templates/verkiezingen/installatievergadering.hbs

```diff
@@ -0,0 +1,74 @@
+{{page-title "Voorbereiding legislatuur"}}
+
+{{#if (or (not @model.isRelevant) (not @model.installatievergadering))}}
+  <div class="au-o-box">
+    <div class="au-o-flow">
+      <AuHeading @skin="2">Voorbereiding legislatuur</AuHeading>
+    </div>
+  </div>
+  <div class="au-o-box">
+    <AuAlert @icon="info-circle" @size="small">
+      <p>De voorbereiding van de legislatuur is niet mogelijk in de
+        bestuurseenheid
+        {{@model.bestuurseenheid.classificatie.label}}
+        {{@model.bestuurseenheid.naam}}.</p>
+      <p>
+        De voorbereiding van de legislatuur gebeurt in de bestuurseenheid
+        Gemeenteraad
+        {{@model.bestuurseenheid.naam}}</p>
+    </AuAlert>
+  </div>
+{{else}}
+  <AuToolbar
+    @size="large"
+    @nowrap="true"
+    class="au-u-padding-bottom-none"
+    as |Group|
+  >
+    <Group>
+      <div class="au-o-flow">
+        <AuHeading @skin="2">Voorbereiding legislatuur</AuHeading>
+      </div>
+    </Group>
+    <Group>
+      <Mandatenbeheer::BestuursperiodenSelector
+        @options={{@model.bestuursPeriods}}
+        @selected={{@model.selectedPeriod}}
+        @onSelect={{this.selectPeriod}}
+        @prefix="Bestuursperiode"
+      />
```

Comment:
Current period is 2025 -heden is ok but when switching to a period in the past and wanting to go back to heden it is not possible

![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/5502b63b-ae73-4963-8be7-3168f85c28a8)

Reply:
this was an issue with the sorting: if an expected organ type is missing, the list of organs contains a null value. fixed that.